doc/classes/AnimationTreePlayer.xml

```diff
@@ -115,6 +116,10 @@
 			</argument>
 			<description>
 				Sets the blend amount of a Blend2 node given its name and value.
+				A Blend2 Node blends two animations with the amount between 0 and 1.
```

Comment:
This paragraph contains multiple capitalizations (**N**ode, **O**utput) that don't feel useful to me.